api/src/main/java/org/arquillian/cube/q/api/Q.java

```diff
@@ -59,10 +59,27 @@ public boolean isExecutable() {
         }
     }
 
-    public static abstract class FloatType {
+    public static abstract class BaseType {
```

Comment:
To create a "DSL" type, to avoid having a method with (Long, Long, Long, Long) what is the first long? and the second? and so on. In this way developer knows every time what is each parameter